added new pump modl and shifted wiyerr conection point douwn

DallinTG opened the pull request:
added new pump modl and shifted wiyerr conection point douwn

FlexibleGames commented:
I edited the model before I committed it so **if** you didn't edit what was there and just overwrote the file all my edits would be lost. 
The wire connection was one of the things I edited as I have to reference it in the JSON. So if it moves more work is required to then change the selection box in JSON.

DallinTG commented:
updated pump shifted wire conecting point moved "WireConnection", to the root nolonger a child changed the textur pathes to 		"pipe": "block/bluepaintedwornsteelplate",
		"cable": "block/cable",
		"castiron": "block/castiron",
		"gold": "block/gold",
		"Mesh": "block/mesh",
		"wornsteel": "block/wornsteel"
and aded 		"collapsedPaths": "Root/PipeConNorth11,Root/center",

DallinTG commented:
fixed the top plate to work with the old pipes fixed some texture overlapping added a ring around the middle plate for contrast and lowered it shifted the pistons to be properly centered and farther out

just added the water bit to
